src/3.1/docker-entrypoint.sh

```diff
@@ -165,6 +165,9 @@ done
 if [[ "$(id -u)" = "0" ]]; then
   chmod -R 755 /data
   chown -R "${userid}":"${groupid}" /data
+  # Chown the log dir also
+  chmod -R 755 /logs
+  chown -R "${userid}":"${groupid}" /logs
```

Comment:
This change could break things for existing users who have a valid setup. if `/logs` is mounted from outside then this will change the owner on the directory on the host machine!

Are we definitely happy to do that? 

I think it would probably be better to check if the user can write to the logs directory and if they can't to exit with an error.

To be honest I think we should do the same with `/data` (i.e. fail fast with an error instead of changing the owner). But since that would break for existing users we can't in `3.x`.

Reply:
I don't think there's a way of creating a writable folder on the host that the `neo4j` user in the container has write permissions to. Other than passing the user/group ids into `docker run` anyway.

Even if I `chown` the folder on the host to `neo4j:neo4j` it doesn't work. Probably because the host user neo4j is different to the container user neo4j. :woman_shrugging: 

I'm not sure what the best solution is. We could chown it but print a warning that we did so?

Reply:
I don't think there's an easy solution to this. The solution with the `chown` works for our data directory and is how redis solve the problem of shared file access from a container: https://github.com/docker-library/redis/blob/master/5.0/Dockerfile

Checking we can write to a folder will always fail, unless container is run with a specific user and group.